lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -18,3 +18,16 @@
     vertical-align: 4px;
   }
 }
+
+.nx-radio-checkbox__input:checked:disabled {
+  + .nx-radio-checkbox__control {
+    &.nx-checkbox__box {
+      background-color:var(--nx-color-interactive-background-disabled);
+      border-color: var(--nx-color-interactive-border-disabled);
+
+      .fa-check {
+        color: var(--nx-color-text-disabled);
+      }
+    }
+  }
+}
```

Comment:
It looks like your editor isn't configured to leave a newline at the end of each file.  We try to do that

Reply:
> It looks like your editor isn't configured to leave a newline at the end of each file. We try to do that

Adjusted VS Code settings and implemented here as well.
92cc934dd7e32144360bfe9850b9403c9b1da9f9